pom.xml

```diff
@@ -4,16 +4,16 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>io.ecocode</groupId>
-    <artifactId>ecocode-php-plugin</artifactId>
+    <artifactId>creedengo-php-plugin</artifactId>
     <version>1.5.2-SNAPSHOT</version>
 
     <packaging>sonar-plugin</packaging>
 
-    <name>ecoCode - PHP language</name>
+    <name>creedengo - PHP language</name>
     <description>Provides rules to reduce the environmental footprint of your PHP programs</description>
     <inceptionYear>2023</inceptionYear>
```

Comment:
please change the year to 2024 ... and execute maven licence command to update all files